core/src/test/java/software/amazon/awssdk/core/util/MimetypesTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.InputStream;
+import java.lang.management.ManagementFactory;
+import java.util.List;
+import javax.management.Attribute;
+import javax.management.AttributeList;
+import javax.management.MBeanServer;
+import javax.management.MBeanServerFactory;
+import javax.management.ObjectName;
+import org.junit.Test;
+
+public class MimetypesTest {
+
+    /** Tests that we aren't leaking file descriptors. */
+    @Test
+    public void testFileDescriptorLeaks() throws Exception {
+        long fd = getOpenFileDecriptorCount();
+        Mimetypes mt = Mimetypes.getInstance();
```

Comment:
Just wondering how valuable or accurate this test is. If something else calls Mimetypes.getInstance() before this test then it wouldn't catch a regression.

Reply:
Yup, I ported this from v1 blindly. Removed this test and added more valuable tests.